### PR TITLE
feat: Add D^3 repo

### PR DESCRIPTION
**Pull request by tumido**
After repo transfer list this repo in the org config

/cc @hemajv 
/assign @goern

### CODE REVIEW

File: github-config.yaml [hunk 1]
```diff
@@ -136,6 +136,10 @@ orgs:
           by our bots...
         has_projects: false
         has_wiki: false
+      data-driven-development:
+        description: ""
```
Comment (hemajv) [inline github-config.yaml:140]: can you please add a small description: `This repository consolidates and provides resources, guides and documentation necessary to foster data driven development.`

**Review by hemajv — CHANGES_REQUESTED** [review]

File: github-config.yaml (hunk 1 above)
Comment (tumido) [reply github-config.yaml]: @hemajv done :+1:

**Comment by tumido** [comment]
/cc @goern 

please review once more

**Pull request merged**